client/src/views/Game.vue

```diff
@@ -227,25 +227,15 @@
         </v-card-actions>
       </v-card>
     </v-dialog>
-
-      <v-snackbar
-        v-model="snackbar"
-        color="info"
-        :timeout='4000'
-        v-show="gameState.status === 'Playing' && playerName !== newMessageName"
-        >{{snackbarText}}
-        <v-btn text @click="snackbar=false">
-          Close
-        </v-btn>
-      </v-snackbar>
-
   </div>
 </template>
 
 <script>
 import unoService from "../services/unoService";
+//import snackService from "../services/snackService";
```

Comment:
Please remove commented out code or add a note explaining why we still need it.

Reply:
Oops. thought I got my comments. Looks like I missed a few